app/src/src/components/common/IndividualWork.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react';
+import { StyleSheet, View, TouchableOpacity, Text } from 'react-native';
+import { IconButton } from 'react-native-paper';
+import toMonthDayYearInWords from '../common/FormatDate/toMonthDayYearInWords';
+// May be used for education/jobs section of edit profile page
+// Basic outline for displaying the individual work of the edit profile page
+const IndividualWork = ({ job, isEdit, setVisible, isLast }) => {
+  // useStates for job data
+  const [company, setCompany] = useState(job.company);
+  const [position, setPosition] = useState(job.position);
+  const [start, setStart] = useState(job.start_date);
+  const [end, setEnd] = useState(job.end_date);
+  const [city, setCity] = useState(job.location);
+  const [info, setInfo] = useState(job.content);
+  const getMonthYear = (date) => {
+    const result = toMonthDayYearInWords({ date: date }).split(' ');
+    return result[1] + ' ' + result[2];
+  };
```

Comment:
This function can live in another file outside this component since it does not depend on any state or props for this component.